### ISSUE TITLE
Scale factor for Image Layer 

### ISSUE THREAD

**Issue by fmoo** [bjorn/tiled#1306]
Is there an easy way to apply a scale factor for an image layer?  Would be nice to not have to double the size of the input image.

**Comment by bjorn** [comment]
There isn't, but if you place your image as a tile object, you can resize it.

Since this is essentially duplication of functionality, I'm hesitant to add image scaling to the image layer.

**Comment by karai17** [comment]
I really have to wonder if the image layer is even a useful feature in Tiled, when simply importing the image as a single tile tileset and tossing it in as an object gives it far, far more functionality and flexibility. Why not just axe the image layer and recommend to people that they do that instead?

**Comment by bjorn** [comment]
Of course, when the image layer was added (with Tiled 0.9, #169), tile objects (added in Tiled 0.6) were a lot more primitive. Yet, even then the image layer didn't add any new functionality. :-/

So yeah, the feature should really just be dropped. I think usability-wise though, some people are using this feature for foreground or background layers. And at the moment using tile objects for these would be very annoying since there is no way to avoid selecting them (short of making their layer invisible). So what I'd like to add first is layer locking, and then I can deprecate image layers because there's no reason left to use them.

**Comment by karai17** [comment]
Sounds good to me.


---
Landon Manning
lmanning17@gmail.com
